.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = src/ExternalModules/CurlSharp
 	url = https://github.com/Sonarr/CurlSharp.git
 	branch = master
+[submodule "src/ExternalModules/SocksWebProxy"]
+	path = src/ExternalModules/SocksWebProxy
```

Comment:
https://www.nuget.org/packages/SocksWebProxy/ is .Net 4.5. 

I forked it to make a .Net 4 version (it doesn't have any 4.5 code, but it breaks, as Sonarr is 4.0)